packages/core/components/DataSourcePrompt/index.tsx

```diff
@@ -136,6 +131,15 @@ export default function DataSourcePrompt(props: Props) {
                                 </li>
                             ))}
                         </ul>
+                        <li className={styles.details}>
+                            Optionally, supply an additional metadata source file to add more
+                            information about the data source. This file should have a header row
+                            column named &quot;Column Name&quot; and another column named
+                            &quote;Description&quot; and a row with the details for each filled in
```

Comment:
`&quote` instead of `&quot`

Reply:
Resolved in https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/183/commits/19f714d39179d6380d457b83107d44ba92e1152b